features/cli/deploy.feature

```diff
@@ -361,6 +362,7 @@ Feature: deployment related features
       | replicas | 1                |
     Then the step should succeed
     Given I wait until number of replicas match "0" for replicationController "ab-example-b-1"
+    Given I wait until number of replicas match "1" for replicationController "ab-example-a-1"
```

Comment:
Also this line, may be it is better to add as below ?

Given number of replicas of "ab-example-a-1" deployment becomes:
      | current | 1 |

Reply:
@zhouying7780  ^^